Formula/conjure-up.rb

```diff
@@ -0,0 +1,92 @@
+class ConjureUp < Formula
+  include Language::Python::Virtualenv
+
+  desc "Big software deployments so easy it's almost magical."
+  homepage "http://conjure-up.io"
+  url "https://github.com/conjure-up/conjure-up/archive/2.1.tar.gz"
+  sha256 "933f94c21b999fd24b4c304ba27c59bde9c83f478f5ee6a45cb8f265aaac2658"
+
+  head "https://github.com/conjure-up/conjure-up.git", :branch => "master"
+
+  devel do
+    url "https://github.com/conjure-up/conjure-up/archive/2.2.0-beta1.tar.gz"
+    version "2.2-beta1"
```

Comment:
Our product follows the same release and version scheme as https://github.com/juju/juju/releases so while not the way you or I would version, it is what I have to work with :)